reciprocalspaceship/dataset.py

```diff
@@ -292,6 +293,53 @@ def _handle_cached_dtypes(dataset, columns, drop):
             dataset = _handle_cached_dtypes(dataset, columns, drop)
             return dataset
 
+    def to_numpy(self, dtype=None, copy=False, na_value=lib.no_default):
+        """
+        Convert the DataSet to a NumPy array.
+
+        By default, the returned array will be `float32` if all columns of the DataSet are
```

Comment:
I find this paragraph hard to read. I took a stab at improving it:

```
This method will attempt to infer a consensus numpy dtype from 
the dtypes of the DataSet columns. If the DataSet is composed of 
all `int32`-backed MTZ dtypes and does contain NaN values, the returned
 `dtype` will be `int32`. For all other combinations of `MTZDtype`, 
the returned dtype will be `float32`.  If the DataSet contains dtypes other 
than `MTZDtype`, the default Pandas behavior is used. 
```

Reply:
Updated to use this language. I also added a link to the Pandas documentation at the end.